api/KeyframeEffect.json

```diff
@@ -128,22 +128,9 @@
                 ]
               }
             ],
-            "firefox_android": [
-              {
-                "version_added": "80"
-              },
-              {
-                "version_added": "63",
-                "version_removed": "79",
-                "flags": [
-                  {
-                    "type": "preference",
-                    "name": "dom.animations-api.compositing.enabled",
-                    "value_to_set": "true"
-                  }
-                ]
-              }
-            ],
+            "firefox_android": {
+              "version_added": "80"
```

Comment:
So now we become a bit inconsistent here. A side affect for making an exception to the 2 year rule that I didn't think about before :( 

If you make different rules for different browsers when to remove irrelevant flags, well then you end up with flags removed for Firefox Android here but not for Firefox Desktop. That looks odd. So maybe we should either not make an exception to the two year rule or we should treat Firefox Desktop with the same exception... What do you think?

Reply:
This is a good point!  I'm leaning towards making the exception for both mobile and desktop, so that the data is consistent between the two.

Reply:
Done!